cloud-common/src/main/java/com/sequenceiq/cloudbreak/common/model/user/CloudbreakUser.java

```diff
@@ -12,11 +12,14 @@
 
     private final String tenant;
 
-    public CloudbreakUser(String userId, String username, String email, String tenant) {
+    private final String crn;
+
+    public CloudbreakUser(String userId, String username, String email, String tenant, String crn) {
```

Comment:
FYI, I've made a similar change [here](https://github.com/hortonworks/cloudbreak/pull/4838) -- we should talk to make sure we don't step on each other. Also want to make sure I'm going about things the right way.

Reply:
Thanks for info. To be honest, I do not know how we can avoid conflicts. I assume both of our branches cannot work without things like crn in cloudbreakuser.

Reply:
Yeah, good point. This may be a necessary side effect high velocity feature changes.